docker/importer/importer.py

```diff
@@ -105,28 +110,32 @@ def checkout(self, source_repo):
 
   def import_new_oss_fuzz_entries(self, repo, oss_fuzz_source):
     """Import new entries."""
-    # TODO(ochang): Make this more efficient by recording whether or not we
-    # imported already in Datastore.
+    exported = []
     vulnerabilities_path = os.path.join(
         osv.repo_path(repo), oss_fuzz_source.directory_path or '')
-    for bug in osv.Bug.query(osv.Bug.status == osv.BugStatus.PROCESSED):
+    for bug in osv.Bug.query(
+        osv.Bug.source_of_truth == osv.SourceOfTruth.INTERNAL):
+      if bug.status != osv.BugStatus.PROCESSED:
```

Comment:
so this cannot be added to query clause above ?

Reply:
Want to avoid adding new indexes. Should be a negligible performance difference.